lib/linter-rust.coffee

```diff
@@ -49,6 +55,9 @@ class LinterRust
         else
           resolve []
 
+      if do @ableToJSONErrors
```

Comment:
@Arcanemagus I'll cache it for a single linter usage (meaning per-`lint` call), but I think it would be incorrect to cache it globally (I mean for a duration of Atom running) -- a user may switch between versions of rustc, while still using the same Atom process running).